### DISCUSSION TITLE
API > Best Performing Songs

### DISCUSSION THREAD

**Discussion by ckeurk** [AzuraCast/AzuraCast#7345] [Q&A] [answered]
Hi, I'd like to know how to get the "Best Performing Songs" info, I can't find it in the API.
Thank you
![bestperformingSongs](https://github.com/user-attachments/assets/d0784925-1bde-43f1-a86c-1624ef5204cc)

**Comment by gAlleb** [comment, marked as answer]
Hi @ckeurk! `/api/station/{station_id}/reports/overview/best-and-worst`

Hint: you can always find such info by inspecting the console during site exploration when hiting a link and looking at a GET request.

<img width="500" alt="image" src="https://github.com/user-attachments/assets/ade86f1b-2071-42a3-9bf7-800235933a66">

<hr>

`https://domain.com/api/station/1/reports/overview/best-and-worst` 
`https://domain.com/api/station/1/reports/overview/best-and-worst?start=***DATE_RANGE***&end=***DATE_RANGE***`

`https://demo.azuracast.com/api/station/1/reports/overview/best-and-worst`

<img width="500" alt="image" src="https://github.com/user-attachments/assets/6a7f9461-dd38-4888-ab47-05c84e8c5831">

**Comment by ckeurk** [comment]
Thank you for your help 😊
